lib/assets/stylesheets/overrides.scss

```diff
@@ -544,3 +541,8 @@ $tooltip-background: $grey;
   border: none;
   background: none;
 }
+
```

Comment:
we should flag changes to overrides.scss in release notes since it means people will need to update them if they keep the customized files in a different repository